gltest_cli/logging.py

```
@@ -0,0 +1,43 @@
+    }
+
+    def format(self, record):
+        if record.levelname in self.COLORS:
+            record.levelname = (
+                f"{self.COLORS[record.levelname]}{record.levelname}{Style.RESET_ALL}"
+            )
+        return super().format(record)
+
+
+def setup_logger():
+    logger = logging.getLogger("gltest")
```

Pylint (3.3.7)

[error] Module 'logging' has no 'getLogger' member

(E1101)